src/components/UI/Header/index.css

```
@@ -129,15 +128,22 @@ option {
   cursor: pointer;
 }
 
-.headerIcons > *:hover {
+.headerIcons > .svg-button {
+  color: white;
```

We should always use variables for the colors.
```suggestion
  color: var(--text-primary);
```